soapui/src/main/java/com/eviware/soapui/support/editor/registry/InspectorRegistry.java

```diff
@@ -32,14 +32,13 @@
     private static InspectorRegistry instance;
     private List<InspectorFactory> factories = new ArrayList<InspectorFactory>();
 

```

Comment:
While you're here, introduce the diamond operator to remove some noise: new ArrayList<>()